packages/lesswrong/server/ckEditor/ckEditorWebhook.ts

```diff
@@ -99,20 +153,52 @@ async function handleCkEditorWebhook(message: any) {
     case "comment.removed":
     case "commentthread.removed":
     case "commentthread.restored":
+    case "collaboration.user.connected": {
+      const userConnectedPayload = payload as CkEditorUserConnectionChange;
+      const userId = userConnectedPayload?.user?.id;
+      const documentId = userConnectedPayload?.document?.id;
+
+      const result = await createMutator({
+        collection: CkEditorUserSessions,
+        document: {
+          userId,
+          documentId
+        },
+        validate: false,
+      });
+
+      break
+    }
     case "document.user.connected":
-      break;
+    case "collaboration.user.disconnected": {
```

Comment:
Did `document.user.disconnected` get renamed to `collaboration.user.disconnected` (as per https://ckeditor.com/docs/cs/latest/developer-resources/webhooks/events.html#user-disconnected)?  Or is there some other difference?

Reply:
Yeah, I'm not sure how come Jim was using`document.user.disconnected`. The API documentation I found used `collaboration.user.disconnected`.

So it's possible we want to rename the other ones too, but would want to chat to @jimrandomh first.